label_sync/main.go

```diff
@@ -423,30 +425,29 @@ func main() {
 type filter func(string, string) bool
 
 func SyncOrg(org string, githubClient *github.Client, config Configuration, filt filter) error {
-	logrus.Infof("Reading repos in %s", org)
+	logrus.WithField("org", org).Info("Reading repos")
 	repos, err := LoadRepos(org, githubClient, filt)
 	if err != nil {
 		return err
 	}
 
-	logrus.Infof("Reading labels in %d repos in %s", len(repos), org)
+	logrus.WithField("org", org).Infof("Found %d repos", len(repos))
 	currLabels, err := LoadLabels(githubClient, org, repos)
 	if err != nil {
 		return err
 	}
 
-	logrus.Infof("Syncing labels for %d repos in %s", len(repos), org)
+	logrus.WithField("org", org).Infof("Syncing labels for %d repos", len(repos))
 	updates, err := SyncLabels(config, *currLabels)
 	if err != nil {
 		return err
 	}
-	if *debug {
-		y, _ := yaml.Marshal(updates)
-		fmt.Println(string(y))
-	}
+
+	y, _ := yaml.Marshal(updates)
+	logrus.Debug(string(y))
 
 	if *dry {
-		logrus.Infof("No real update labels in --dry-run")
+		logrus.Infof("Running in --dry-run mode, no mutations made")
```

Comment:
can we switch this to `--wet`? 😈

Reply:
Not sure I understand -- would you like me to change the flag name>

Reply:
I'm mostly joking, although I'd rather have `--foo` flags than `--unfoo=false` (thus `--wet` rather than `--dry-run=false`)

Reply:
Yeah, I did get tripped up by `--dry-run=false`. I'll tackle that in a separate PR.